pkg/opensearch/time_series_query_test.go

```diff
@@ -1318,3 +1318,133 @@ func TestSettingsCasting_luceneHandler_processQuery_processLogsQuery_ignores_any
 		Offset:         "",
 	}, sr.Aggs[0].Aggregation.Aggregation.(*es.DateHistogramAgg))
 }
+
+func Test_trace_list(t *testing.T) {
```

Comment:
This is a go typed version of the query building. It is similar to the snapshot test, but the snapshot test is more "whole" and "complete."

It may be easier to debug differences in objects in this test than in the snapshot test. Even though it is a duplicate, I would keep it since it is already written. It also matches the style of the other tests for other query types.